_submitplugins/rrSubmit_Nuke_5.py

```diff
@@ -246,6 +246,11 @@ def submitJobsToRR(jobList, submitOptions, nogui=False):
         commandline=getRRSubmitterconsolePath() + "  \"" + tmpFile.name + "\"" + ' -Wait'
     else:
         commandline= getRRSubmitterPath()+"  \""+tmpFile.name+"\""
+
+    if own_terminal:
+        if sys.platform.lower().startswith("win"):
+            commandline = 'START CMD /C "{0}"'.format(commandline)
```

Comment:
Maybe the problem here is that you have quotes around `{0}` causing the actual command to have double, or inverted quotes.
Try without quotes but with a title:
`commandline = 'START "rrSubmitter" CMD /C {0}'.format(commandline)`

Reply:
I had misunderstood how START works. Please check my latest commit

Reply:
Looks good to me if it works for you.